public/res/default.json

```diff
@@ -66,6 +67,35 @@
       "button": "Read more"
     }
   },
+  "home_page": {
+    "title_line1": "<0>Data discovery</0> and <2>access</2>",
+    "title_line2": "for <1>large-scale</1> science facilities",
+    "browse": {
+      "title": "Browse, explore and visualise experimental data",
+      "description1": "Large scale facilities, such as synchrotrons, neutron and muon sources, lasers and accelerators, generate vast amounts of data that need to be managed in an efficient way, supporting data ingestion for long-term storage and archival, as well as data analysis and data publication workflows.",
+      "description2": "<0>DataGateway</0> focuses on providing data discovery and data access functionality to the data.",
+      "link": "/browse/investigation",
+      "button": "Browse data"
+    },
+    "search": {
+      "title": "Search",
+      "description": "Search for the experimental data according to different criteria.",
+      "link": "/search/data",
+      "button": "Search data"
+    },
+    "download": {
+      "title": "Download",
+      "description": "Retrieve the experimental data using a variety of download methods.",
+      "link": "/download",
+      "button": "Download data"
+    },
+    "facility": {
+      "title": "ISIS Neutron and Muon Source",
+      "description": "World-leading centre for research giving unique insights into the properties of materials on the atomic scale.",
+      "link": "https://www.isis.stfc.ac.uk/Pages/About.aspx",
+      "button": "Read more"
+    }
+  },
```

Comment:
Not sure how this happened... did you accidentally copy this over from somewhere?

Reply:
Oops, I didn't realise I didn't revert my default.json before modifying it - I'll fix it now...